README.md

```diff
@@ -172,82 +100,92 @@ In Device Simulator Express, you can use keyboard to interact with the device:
 
-1. Plug in your BBC micro:bit.
-2. Run the command `"Device Simulator Express: Open Serial Monitor"`.
-3. Select your baud rate for the serial port.
-4. The `print()` statements in your code will show in the output console.
+- *For the micro:bit*:
+    - Download the firmware with the .hex file (link: https://microbit.org/get-started/user-guide/firmware/).
 
-#### VII. Use the sensors in the Device Simulator Express
+1. Plug in your device (make sure it’s formatted properly already).
+2. Run the command `"Device Simulator Express: Deploy to Device"`.
 
-Generating input for the sensors can be done by interacting directly with device on the webview
-or by using the toolbar.
+<img alt='Deploy to Device' src='assets/readmeFiles/deploy.png'>
 
-- **Push buttons:** click directly on the buttons on the device or use the keybindings.
-- **Temperature sensor, Light sensor, Acceleration sensor:** click on the corresponding button in the toolbar and change the value using the slider or the input box attached to it.
+### VI. Use the Serial Monitor for your device (available on Windows and Mac only)
+
+1. Plug in your device (make sure it’s formatted properly already).
+2. Run the command `"Device Simulator Express: Open Serial Monitor"`.
+3. Select your baud rate for the serial port.
+4. The `print()` statements in your code will show in the output console.
 
-#### VIII. Debug your project on the simulator
+### VII. Debug your project on the simulator
 
 1. Add breakpoints in your code
 2. Press F5 to enter the debugging mode, and you can start debugging line by line!
 
 ### Commands
-Using the simulator for the micro:bit is similar to using the one for the CPX. The only difference is that the commands in the command palette display `Device Simulator Express: [micro:bit] <command>` instead of `Device Simulator Express: [Circuit Playground Express] <command>`. Currently, we support the following commands for micro:bit:
+
+Device Simulator Express provides several commands in the Command Palette (`F1` or `CTRL + SHIFT + P` / `CMD + SHIFT + P` for Mac OS) for working with \*.py files:
+
 - `Device Simulator Express: Getting Started`: Opens a page in VS Code that helps users get started with the extension. Here, users can browse through code that they can use to play with the simulators.
 - `Device Simulator Express: Run Simulator`: Runs Python code on the simulator.
-- `Device Simulator Express: [micro:bit] Open Simulator`: Opens an unsaved .py file with template code, also opens the simulator.
-- `Device Simulator Express: [micro:bit] New File`: Opens the simulator in the webView.
-- `Device Simulator Express: [micro:bit] Deploy to Device`: Copies the current file to the micro:bit if the device is detected.
+- `Device Simulator Express: New File`: Opens an unsaved .py file with template code, also opens the simulator for the selected device.
+- `Device Simulator Express: Open Simulator`: Opens the simulator in the webview for the selected device

```

Comment:
nit: I wouldn't use the word webview, but more right-side window.

Reply:
Very good point. However, the window can be on the left side, so I'll say "simulator window". What do you think?

Reply:
That works!